core/testdata/config-full.toml

```diff
@@ -0,0 +1,272 @@
+ServerPubKey = 'test-pub-key'
+URL = 'https://prom.test'
+BufferSize = 1234
+MaxBatchSize = 4321
+SendInterval = '1m0s'
+SendTimeout = '5s'
+UseBatchSend = true
+
+[Log]
+JSONConsole = true
+FileDir = 'log/file/dir'
+SQL = true
+FileMaxSize = '100.00gb'
+FileMaxAge = 17
+FileMaxBackups = 9
+UnixTS = true
+
+[WebServer]
+AllowOrigins = '*'
+AuthenticatedRateLimit = 42
+AuthenticatedRateLimitPeriod = '1s'
+BridgeResponseURL = 'https://bridge.response'
+HTTPWriteTimeout = '1m0s'
+Port = 56
+SecureCookies = true
+SessionTimeout = '1h0m0s'
+UnAuthenticatedRateLimit = 7
+UnAuthenticatedRateLimitPeriod = '1m0s'
+RPID = 'test-rpid'
+RPOrigin = 'test-rp-origin'
+TLSCertPath = 'tls/cert/path'
+TLSHost = 'tls-host'
+TLSKeyPath = 'tls/key/path'
+TLSPort = 6789
+TLSRedirect = true
+
+[JobPipeline]
+DefaultHTTPLimit = 67
+DefaultHTTPTimeout = '1m0s'
+FeatureExternalInitiators = true
+MaxRunDuration = '1h0m0s'
+ReaperInterval = '4h0m0s'
+ReaperThreshold = '168h0m0s'
+ResultWriteQueueDepth = 10
+
+[OCR2]
+ContractConfirmations = 11
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+ContractTransmitterTransmitTimeout = '1m0s'
+DatabaseTimeout = '8s'
+KeyBundleID = 'test-bundle-id'
+MonitoringEndpoint = 'test-mon-end'
+
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+DefaultTransactionQueueDepth = 12
+KeyBundleID = 'test-key-bundle-id'
+MonitoringEndpoint = 'test-monitor'
+SimulateTransactions = true
+TransmitterAddress = '0x1234abcd'
+OutgoingMessageBufferSize = 7
+IncomingMessageBufferSize = 3
+DHTLookupInterval = 9
+BootstrapCheckInterval = '1m0s'
+NewStreamTimeout = '1s'
+
+[P2P]
+NetworkingStack = 'V1V2'
+IncomingMessageBufferSize = 13
+OutgoingMessageBufferSize = 17
+AnnouncePort = 1234
+BootstrapCheckInterval = '1m0s'
+BootstrapPeers = ['foo', 'bar', 'should', 'these', 'be', 'typed']
+DHTAnnouncementCounterUserPrefix = 4321
+DHTLookupInterval = 9
+ListenPort = 9
+NewStreamTimeout = '1s'
+PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw'
+PeerstoreWriteInterval = '1m0s'
+V2AnnounceAddresses = ['a', 'b', 'c']
+V2Bootstrappers = ['1', '2', '3']
+V2DeltaDial = '1m0s'
+V2DeltaReconcile = '1s'
+V2ListenAddresses = ['foo', 'bar']
+
+[Keeper]
+CheckUpkeepGasPriceFeatureEnabled = true
+DefaultTransactionQueueDepth = 17
+GasPriceBufferPercent = 12
+GasTipCapBufferPercent = 43
+BaseFeeBufferPercent = 89
+MaximumGracePeriod = 31
+RegistryCheckGasOverhead = 90
+RegistryPerformGasOverhead = 18446744073709551615
+RegistrySyncInterval = '1h0m0s'
+RegistrySyncUpkeepQueueSize = 31
+TurnLookBack = 91
+TurnFlagEnabled = true
+
+[AutoPprof]
+Enabled = true
+ProfileRoot = 'prof/root'
+PollInterval = '1m0s'
+GatherDuration = '12s'
+GatherTraceDuration = '13s'
+MaxProfileSize = '1.00gb'
+CPUProfileRate = 7
+MemProfileRate = 9
+BlockProfileRate = 5
+MutexProfileFraction = 2
+MemThreshold = '1.00gb'
+GoroutineThreshold = 999
+
+[[EVM]]
```

Comment:
https://toml.io/en/v1.0.0#array-of-tables

Alternatively, an earlier version of this PR used maps keyed on chain id and node name, which seems natural since we want those to be unique anyways, but in practice the extra level or two of nested keys was messy and obscure w/o field names, and has to be duplicated in all of the sub-tables too.